content/docs/user-guide/integrations/databricks.md

```diff
@@ -0,0 +1,89 @@
+# Databricks
+
+As of September 2023 Databricks doesn't expose the underlying GIT repo in your
+project, so GIT-related DVC functionality within the repo provided by Databricks
+is not supported (e.g. [experiments], `--rev/--all-commits/--all-tags/etc`). But
+everything will operate as normal if you `git clone` a project yourself or use
+remote projects with DVC directly.
+
+## Setup
```

Comment:
This doesn't apply to the dvc api example, right? Should we move it down or clarify that that there are two different workflows here? Maybe we should divide by working with a data registry vs working inside a dvc repo instead of API vs commands?

Reply:
@dberenbaum To the example itself no, but it apples to the API if you'll use `.` as a target, so I decided to keep it this way.

Reply:
> Maybe we should divide by working with a data registry vs working inside a dvc repo instead of API vs commands?

Thought about it, but felt strange, because those are the same cli commands and python API that can both accept remote urls and current repo.